test/integration/docker_save_test.js

```diff
@@ -47,10 +48,12 @@ suite('use docker-save', () => {
 
     let url = `https://queue.taskcluster.net/v1/task/${taskId}/runs/${runId}/artifacts/public/dockerImage.tar`;
 
-    //superagent means no unzipping required
+    //superagent means no zlib required
     let res = await request.get(url).end();
     res.pipe(fs.createWriteStream('/tmp/dockerload.tar'));
     await waitForEvent(res, 'end');
+    //make sure it finishes unzipping
+    await base.testing.sleep(1000);
```

Comment:
Is there some kind of race from superagent emitting the 'end' event and the file being available?